### PR TITLE
[risk=low] [Db 329] adding medline plus link when search is active

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. Adding medline plus link when search is active.
2. Changing chart interval to display <= 20 for aynthing below 20.

### CODE REVIEW

File: public-ui/src/app/views/ehr-view/ehr-view.component.css [hunk 1]
```diff
@@ -17,19 +17,25 @@
...
+    color: black;
 }
 
 #domain-summary {
     font-size: 23px;
     margin-bottom: 28px;
 }
 
-.domain-info-container {
+.domain-info-layout {
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/ehr-view/ehr-view.component.css:36]: better var

File: public-ui/src/app/views/ehr-view/ehr-view.component.html [hunk 2]
```diff
@@ -43,19 +43,22 @@ <h5 id="domain-summary" class="secondary-display">
           </div>
         </section>
         <div class="results-grid" *ngIf="items.length > 0">
-          <div class="domain-info-container">
+          <div class="domain-info-layout">
+            <span>
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/ehr-view/ehr-view.component.html:37]: added <span> to combine elements into one child  of the flex-container

File: public-ui/src/app/views/ehr-view/ehr-view.component.html [hunk 3]
```diff
@@ -67,6 +70,8 @@ <h5 id="domain-name" class="primary-display" *ngIf="searchText.value.length > 0"
                 </clr-tooltip>
               </h5>
             </ng-template>
+          </span>
+          <h6 *ngIf="searchText.value.length > 0" class="primary-display medline-link">Interested in general health information related to "{{searchText.value}}"? &nbsp; <a href="{{medlinePlusLink}}" target="_blank"> Search Medline Plus</a>.</h6>
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/ehr-view/ehr-view.component.html:64]: if value than show

**Review by mhigbyflowers — APPROVED** [review]
image

File: public-ui/src/app/views/ehr-view/ehr-view.component.html (hunk 3 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/views/ehr-view/ehr-view.component.html:64]: I was just fixing that. :)

Comment (SrushtiGangireddy) [reply public-ui/src/app/views/ehr-view/ehr-view.component.html:64]: thanks.

**Pull request merged**